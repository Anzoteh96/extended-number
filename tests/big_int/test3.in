6369414947
986238439
