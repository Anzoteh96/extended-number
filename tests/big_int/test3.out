Enter two numbers n and m:
n is 6369414947
m is 986238439
n + m is 7355653386
n - m is 5383176508
n * m is 6281761854672547733
n / m is 6
n % m is 451984313
